redfish-core/lib/hypervisor_system.hpp

```diff
@@ -436,27 +518,95 @@ inline void parseInterfaceData(
     jsonResponse["DHCPv4"]["DHCPEnabled"] =
         translateDHCPEnabledToBool(ethData.DHCPEnabled, true);
 
+    jsonResponse["DHCPv6"]["OperatingMode"] =
+        translateDHCPEnabledToBool(ethData.DHCPEnabled, false) ? "Enabled"
```

Comment:
We don't use the Ternary Operator (?) in upstream. I won't enforce but an FYI

Reply:
Done